src/components/ComboButton/ComboButton.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * @file Combo button.
+ * @copyright IBM Security 2019
+ */
+
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React, { Fragment, useState, useRef } from 'react';
+import { ChevronDown16, ChevronUp16 } from '@carbon/icons-react';
+import { settings } from 'carbon-components';
+
+import Button from '../Button';
+import OverflowMenu from '../OverflowMenu';
+import OverflowMenuItem from '../OverflowMenuItem';
+import { TooltipDirection } from '../IconButton/IconButton';
+
+import { namespace as buttonNamespace } from '../Button/Button';
+
+import { getComponentNamespace } from '../../globals/namespace';
+
+export const namespace = getComponentNamespace('combo-button');
+
+const { prefix } = settings;
+
+const OVERFLOW_MENU_ICON = {
+  OPEN: ChevronUp16,
+  CLOSE: ChevronDown16,
+};
+
+const ComboButton = props => {
+  const [overflowMenuIcon, setOverflowMenuIcon] = useState(
+    OVERFLOW_MENU_ICON.OPEN
+  );
+  const wrapper = useRef(null);
+  const { className, overflowMenuDirection, actions } = props;
+  const [firstAction, ...restActions] = actions;
+  const isTop = overflowMenuDirection === TooltipDirection.TOP;
+
+  const getMenuOffset = () => {
+    const { top } = wrapper.current.getBoundingClientRect();
+    return {
+      top: isTop ? top : top * -1,
+      left: 'auto',
+    };
+  };
+
+  const renderActions = items => {
+    const itemClassName = `${prefix}--list-box__menu-item`;
+    return items.map((item, index) => (
+      <OverflowMenuItem
+        className={itemClassName}
+        key={item.label}
+        itemText={
+          <Fragment>
+            <span className={`${itemClassName}-label`} title={item.label}>
+              {item.label}
+            </span>
+            <item.renderIcon className={`${itemClassName}-icon`} />
```

Comment:
Should the icon have an empty description?

Reply:
@diego-codes I think it should be good since `iconDescription` no longer has a default here. So nothing extra gets added to the svg and it keeps `aria-hidden="true"`.

But I did go ahead and remove that `title` from the `span` because it doesn't seem necessary. 🤔 What do you think about it now?

Reply:
I think the `title` is necessary for when the text overflows out of the allowed space so it is truncated with an ellipsis.

Reply:
oh oops, you are right -- will add that back 👍

Reply:
Hey one thing about this: now that we're using `ComboButtonItem`, which is basically just a renamed `OverflowMenuItem` that allows users to pass in any old node via the `itemText` prop, they would likely need to control this truncation themselves 🤔 I added an example & notes to the storybook on how to do that though.